src/pollution_model/run_pollution_model.py

```diff
@@ -0,0 +1,454 @@
+        duration of the rainfall event (h)
+    surface_type: int
+        the type of surface we are computing the TSS for
+
+    Returns
+    -------
+    float
+       Returns the TSS value from the given parameters
+    """
+
+    # Define the constants (Cf is the capacity factor).
+    # Values a1 to a3 are empirically derived coefficient values.
+    capacity_factor = 0.25 if surface_type == 1 else 0.75
+    a1, a2, a3 = 0, 0, 0
+    match surface_type:
+        case "Cr":
+            a1, a2, a3 = 0.6, 0.25, 0.00933
+        case "Cu":
+            a1, a2, a3 = 2.5, 0.95, 0.00933
+        case "Gv":
+            a1, a2, a3 = 0.6, 0.5, 0.00933
+        case "Rd" | "CrP":
+            a1, a2, a3 = 2.9, 0.16, 0.0008
+        case _:
+            log.error(
+                f"Given surface is not valid for computing TSS. Got {SurfaceType(surface_type).name}.")
+            raise ValueError(f"Given surface type is not valid for computing total suspended solids. Needed a roof or "
+                             f"road, but got {SurfaceType(surface_type).name}.")
+    first_term = surface_area * a1 * (antecedent_dry_days ** a2) * capacity_factor
+    second_term = (1 - math.exp(a3 * average_rain_intensity * event_duration))
+
+    return first_term * second_term
+
+
+def total_metal_load_roof(surface_area,
+                          antecedent_dry_days,
+                          average_rain_intensity,
+                          event_duration,
+                          rainfall_ph,
+                          surface_type):
+    """
+    Calculates the total metal load for a given roof;
+
+    Parameters
+    ----------
+    surface_area: float
+        surface area of the given surface type
+    antecedent_dry_days: float
+        length of antecedent dry period (days)
+    average_rain_intensity: float
+        average rainfall intensity of the event (mm/h)
+    event_duration: float
+        duration of the rainfall event (h)
+    rainfall_ph: float
+        the acidity level of the rainfall
+    surface_type: int
+        the type of roof we are calculating the metal load for. Some coefficients depend on this.
+
+    Returns
+    -------
+    (float, float)
+       Returns the total copper and zinc loads from the given parameters (micrograms)
+    """
+    # Define constants in a list
+    b = []
+    c = []
+
+    match surface_type:
+        case "Cr":
+            b = [2, -2.8, 0.5, 0.217, 3.57, -0.09, 7, -3.73]
+            c = [50, 2600, 0.1, 0.01, 1, -3.1, -0.007, 0.056]
+        case "Cu":
+            b = [100, -2.8, 1.372, 0.217, 3.57, -1, 275, -3.3]
+            c = [-0.1, 2, 0.1, 0.01, 0.8, -1.3, -0.007, 0.056]
+        case "Gv":
+            b = [2, -2.8, 0.5, 0.217, 3.57, -0.09, 7, -3.73]
+            c = [910, 4, 0.2, 0.09, 1.5, -2, -0.23, 1.990]
+        case _:
+            log.error(
+                f"Given surface is not valid for computing total metal load. Got {SurfaceType(surface_type).name}.")

```

Comment:
I think this string shouldn't be duplicated, assign a variable so we don't have to change it in both error messages. Same with in other places in the PR